lib/nostrum/api.ex

```diff
@@ -1355,54 +1360,82 @@ defmodule Nostrum.Api do
   end
 
   @doc """
-  Gets a user.
+  Gets a `Nostrum.Struct.User` by its `user_id`.
```

Comment:
For this I was actually referring to the `user_id` type portion.

Reply:
Would something like this be what we are looking for?

```Elixir
@doc """
Gets a user by its `t:Nostrum.Struct.User.id/0`.
"""
```

Reply:
Yeah that'd work, could we use the alias'd version of the User module? I'm not sure if it works for types. 

Also it'd be nice to add this elsewhere when applicable.

Reply:
Fixed in latest commits. `mix docs` wouldn't compile the documentation properly if I used an alias.